olatundesleek/capcoinfx-fr
# #101 feature(transanction):fetch user transactions

#### What does this PR do?
Fetch transactions 
#### Description of Task to be completed?
- fetch all transactions on a user's account
- format monetary values to be rendered in the USD
#### How should this be manually tested?
- clone the repository'
- cd into the project directory
- run `npm install`
- run `npm start` to start the development server
- login in with the following credentials
email: `admin@test.com`
password: `helloworld`
This will take you to the dashboard area if successfully authenticated
- once signed in, navigate to the `transactions` page to see all transactions on the account
#### Any background context you want to provide?
The local server runs at `http://localhost:3000`
#### What are the relevant pivotal tracker stories?
#101 #94 
#### Screenshots (if appropriate)
n/a
#### Questions:
n/a
